ouroboros-consensus-shelley-test/test/Test/ThreadNet/Shelley.hs

```diff
@@ -114,15 +114,6 @@ instance Arbitrary NightlyTestSetup where
   arbitrary = do
       setup <- arbitrary
 
-      -- At time of writing, this causes 100 tests to have an expected run time
```

Comment:
I'm not sure this comment is no longer relevant. It explains why we chose the frequency numbers. @nfrisby @nc6 thoughts?

Reply:
Yep, I think this information would help future time tuning. @craigem would you explain why the removal? Thanks

Reply:
This was removed due to referencing buildkite which this PR removes,

Happy to add it back in.

Reply:
Yeah, I think it's wise to leave this comment. If we want to remove the reference to Buildkite maybe we can rephrase this to something like:

```
of half an hour on a CI machine.
```

but I don't know if that's any better.

Reply:
Will do that push it back.

Reply:
Re-added in the most recent commit.

Reply:
Thanks Craige. I'm just afraid that "the queue=bench machine" is meaningless. That's why I suggested referring to a CI machine, or maybe keeping the reference to buildkite:

```
This caused 100 tests to have an expected run time of half an hour in a Buildkite machine. Note that the Buildkite CI infrastructure is now deprecated [in favour of ...]. 

```

or something like that.

Reply:
I hope to have address this in the most recent push.

Reply:
Looks good. Thank you Craige :pray: